ui/test/selection_service_test.cpp

```
@@ -0,0 +1,172 @@
+  CHECK(!sel.preselection().has_value());
+  CHECK_EQ_INT(sel.count(), 0);
+  CHECK(!sel.focus().has_value());
+  CHECK_EQ_INT(sel.committed().size(), 0);
+
+  // ── PRESELECTION does not disturb selection, focus or committed ─────────
+  sel.setPreselection(edge("E_hover"));
+  CHECK(sel.preselection().has_value());
+  CHECK_EQ_STR(sel.preselection()->persistentName, "E_hover");
+  CHECK_EQ_INT(sel.count(), 0);          // hovering picked nothing
+  CHECK(!sel.focus().has_value());
+  CHECK_EQ_INT(sel.committed().size(), 0);
```

Cppcheck (2.21.0)

[style] The function 'checkLeInt' is never used.

(unusedFunction)